### PR TITLE
Improve coop ui presentation

### PR DESCRIPTION
**Pull request by Brutus5000**

### CODE REVIEW

File: src/main/java/com/faforever/api/data/domain/CoopScenario.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  public String getName() {
+    return name;
+  }
+
+  @Column(name = "description")
+  public String getDescription() {
+    return description;
+  }
+
+  @OneToMany(mappedBy = "map", fetch = FetchType.LAZY)
```
Comment (bukajsytlos) [inline src/main/java/com/faforever/api/data/domain/CoopScenario.java]: mappedBy = "scenario"?

**Review by bukajsytlos — CHANGES_REQUESTED** [review]

**Review by bukajsytlos — DISMISSED** [review]
lgtm beside missing tests

**Comment by Brutus5000** [comment]
> lgtm beside missing tests

I usually only test Elide things if permissions or business logic is involved. This is default read only.

**Comment by Brutus5000** [comment]
Do we need to setup dedicated caching? Coop usually doesn't change a lot.

**Pull request merged**